File: aat-lib/src/main/java/ch/bailu/aat_lib/service/cache/ObjBroadcastReceiver.java
package ch.bailu.aat_lib.service.cache;

import ch.bailu.aat_lib.app.AppContext;

public interface ObjBroadcastReceiver {
    void onDownloaded(String id, String url, AppContext appContext);
    void onChanged(String id, AppContext appContext);
}

File: aat-lib/src/main/java/ch/bailu/aat_lib/service/cache/ObjBroadcastReceiver.kt
package ch.bailu.aat_lib.service.cache

import ch.bailu.aat_lib.app.AppContext

interface ObjBroadcastReceiver {
    fun onDownloaded(id: String, url: String, appContext: AppContext)
    fun onChanged(id: String, appContext: AppContext)
}
